migrations.json

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
I'm not familiar with this, but I assume NX keeps track which migrations need to run based on this list, that's why we include it to our repo?

Reply:
Yes, it runs the migrations based on the version from/to you're using. We probably don't need to keep it though